lib/execution/runner.js

```diff
@@ -127,9 +128,10 @@ class Runner {
   // to run in sequence, and on the same connection, especially helpful when schema building
   // and dealing with foreign key constraints, etc.
   async query(obj) {
-    const { __knexUid, __knexTxId } = this.connection;
+    const connection = this.connection;
```

Comment:
Here, `this.connection` was used in a few places, but with asynchronous "gaps" between usages. It's technically possible that its value could change between accesses, so I capture it once and reuse it since it makes sense to expect that we're referring to the same "connection" object throughout this method. The tests don't suggest it's breaking anything.